src/macaron/slsa_analyzer/checks/build_as_code_check.py

```diff
@@ -75,13 +84,22 @@ def _has_deploy_command(self, commands: list[list[str]], build_tool: BaseBuildTo
             if not cmd_program_name:
                 logger.debug("Found invalid program name %s.", com[0])
                 continue
-            if any(build_cmd for build_cmd in build_tool.builder if build_cmd == cmd_program_name):
-                # Check the arguments in the bash command for the deploy goals.
-                # If there are no deploy args for this build tool, accept as deploy command.
```

Comment:
Why are you removing these comments?

Reply:
That was an accident :sweat_smile:.

Reply:
Restored these comments in [9c167f2](https://github.com/oracle-samples/macaron/pull/67/commits/9c167f271c3b880c3db95d6de5fc7bb51bdfcf3d).